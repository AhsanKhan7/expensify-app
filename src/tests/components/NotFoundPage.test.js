import React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../../components/NotFound'

test('should not found component run', () => {
    const wrapper =shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
})