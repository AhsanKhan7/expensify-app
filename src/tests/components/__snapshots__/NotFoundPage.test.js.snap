// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not found component run 1`] = `
<div>
  404! - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
